setup.py

```diff
@@ -44,7 +44,7 @@ def get_packages(package):
 env_marker_below_38 = "python_version < '3.8'"
 
 minimal_requirements = [
-    "click==7.*",
+    "click>=7.*",
```

Comment:
I wonder if this should also specify `<9` to avoid an accidental upgrade to Click 9.0 some time in the future.

Reply:
We have time I hope